src/constants/keys.py

```diff
@@ -27,7 +27,7 @@
 # these let us specify the stage specs
 IMAGE = "image"
 ENV = "env"
-ENTRY_POINT = "entry_point"
+ENTRY_POINT = "entrypoint"
```

Comment:
When had you branched off? This seems to be a change already merged into the `master` branch to make it compliant with chainlink.

https://github.com/illinois-cs241/broadway-api/blob/master/src/constants/keys.py#L34

Reply:
Looks like some changes are leading to conflicts with master.

Reply:
my bad. should be fixed now